lambda/wirelessLogic/testData/activeSimHistory2.json

```
@@ -0,0 +1,64 @@
+{
+  "sims": [
+    {
+      "iccid": "89444611503500006596",
+      "msisdn": "883190600377978",
```

Do not include real ICCIDs, use clearly fake ones, e.g.

```suggestion
      "iccid": "89457366600022312493",
      "msisdn": "<random number>",
```

Also, no need to include this many example entries for tests. Only include examples for different test cases.